thrall/test/lib/elasticsearch/ElasticSearchTest.scala

```diff
@@ -15,6 +19,30 @@ class ElasticSearchTest extends ElasticSearchTestBase {
 
     "images" - {
 
+      "bulk inserting" - {
+        "can bulk insert images" in {
+          val imageOne = createImage("batman", StaffPhotographer("Bruce Wayne", "Wayne Enterprises")).copy(
+            userMetadata = Some(Edits(labels = List("foo", "bar"), metadata = ImageMetadata(description = Some("my description"))))
+          )
+
+          val imageTwo = createImage("superman", StaffPhotographer("Clark Kent", "Kent Farm")).copy(
+            usages = List(usage())
+          )
+
+          val images: List[Image] = List(imageOne, imageTwo)
+
+          def eventualMatchAllSearchResponse: Future[Response[SearchResponse]] = ES.client.execute(ElasticDsl.search(elasticSearchConfig.alias).size(images.length * 10))
```

Comment:
Would be nice to re-use `ES.countImages` here, but couldn't get the test to pass. This implementation is inspired by a similar scenario in [`media-api`](https://github.com/guardian/grid/blob/32d06a9db2ff068c48e3a785c09102842d13b5cd/media-api/test/lib/elasticsearch/ElasticSearchTest.scala#L69)